src/widgets/WalletModal/CopyToClipboard.tsx

```diff
@@ -30,17 +30,33 @@ const Tooltip = styled.div<{ isTooltipDisplayed: boolean }>`
 const CopyToClipboard: React.FC<Props> = ({ toCopy, children, ...props }) => {
   const [isTooltipDisplayed, setIsTooltipDisplayed] = useState(false);
 
+  const copyToClipboard = (content: string) => {
+    const el = document.createElement("textarea");
+    el.value = content;
+    document.body.appendChild(el);
+    el.select();
+    document.execCommand("copy");
+    document.body.removeChild(el);
+  };
+
+  function displayTooltip() {
+    setIsTooltipDisplayed(true);
+    setTimeout(() => {
+      setIsTooltipDisplayed(false);
+    }, 1000);
+  }
+
   return (
     <StyleButton
       small
       bold
       onClick={() => {
-        if (navigator.clipboard) {
+        if (document.queryCommandSupported("copy")) {
+          copyToClipboard(toCopy);
+          displayTooltip();
+        } else if (navigator.clipboard) {
           navigator.clipboard.writeText(toCopy);
-          setIsTooltipDisplayed(true);
-          setTimeout(() => {
-            setIsTooltipDisplayed(false);
-          }, 1000);
+          displayTooltip();
```

Comment:
1. This should be the fallback only for WebViews, iFrames and weird browsers, not the primary way to copy the address, not to mention it is (getting) [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/queryCommandSupported) so it's a bit icky to prioritize it.
`navigator.clipboard` check should be first. And since technically Trust Wallet browser has `navigator.clipboard` defined, it fails because its a WebView where `navigator.permissions` is `undefined` (thus not having permission to copy), so we should check for that too. 
2.  I know that it "works" as it is right now, but technically `navigator.clipboard.writeText(toCopy)` returns a promise, so we'd better handle it appropriately. Otherwise uses would see "Copied!" tooltip even if it didn't actually copied anything. Would be also nice to somehow handle error in `.catch()` but I guess (a) unless user uses extremely excotic browser it shouldn't happen (b) showing user "sorry we couldn't copy the address, browser is poop" is questionable 😄 

```suggestion
        if (navigator.clipboard && navigator.permissions) {
          navigator.clipboard.writeText(toCopy).then(() => displayTooltip());
        } else if (document.queryCommandSupported("copy")) {
          copyToClipboard(toCopy);
          displayTooltip();
        }
```